app/src/main/java/org/mozilla/tv/firefox/settings/SettingsRepo.kt

```diff
@@ -0,0 +1,41 @@
+package org.mozilla.tv.firefox.settings
+
+import android.app.Application
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import android.content.SharedPreferences
+import android.os.StrictMode
+import android.preference.PreferenceManager
+import org.mozilla.tv.firefox.R
+
+private const val PREF_KEY_TELEMETRY = R.string.pref_key_telemetry
+
+class SettingsRepo(private val applicationContext: Application) {
+    private val _sharedPreferences: SharedPreferences = PreferenceManager.getDefaultSharedPreferences(applicationContext)
+    private val _resources = applicationContext.resources
```

Comment:
Nit: normally starting underscores are used for backing properties, so suggest removing them here.
https://kotlinlang.org/docs/reference/coding-conventions.html#names-for-backing-properties